### PR TITLE
Add columns option for InfoList component

### PR DESCRIPTION
**Pull request by xwatkins**
Use css columns if a flag is specified as a prop and the screen is bigger than `small`

### CODE REVIEW

File: src/app/components.jsx [hunk 1]
```diff
@@ -182,6 +182,10 @@ const components = [
           title: 'Another item',
           content: <div>Some more content</div>,
         },
+        {
+          title: 'Yet another item',
+          content: loremIpsum({ count: 25, units: 'words' }),
```
Comment (dlrice) [inline src/app/components.jsx]: `content` is a string while the prop type is `PropTypes.element`.

File: src/components/info-list.jsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 
 import '../styles/components/info-list.scss';
 
-const InfoList = ({ infoData }) => (
-  <div className="info-list">
+const InfoList = ({ infoData, columns = false }) => (
```
Comment (dlrice) [inline src/components/info-list.jsx]: The default value for `columns` is specified in the function parameters and also in `defaultProps`. Would be good to stick to one place.

File: src/styles/components/info-list.scss [hunk 3]
```diff
@@ -24,6 +28,23 @@
...
+      }
+    }
+  }
+}
+
+@include breakpoint(medium) {
+  .info-list--columns {
+    column-count: 2;
+    column-width: 50%;
+    .info-list__item {
```
Comment (dlrice) [inline src/styles/components/info-list.scss]: Is `item` a modifier of `columns` or just a different element?

Comment (xwatkins) [reply src/styles/components/info-list.scss]: `columns` is the modifier of the top level element, `item` doesn't change

**Pull request merged**